spec/dummy/config/application.rb

```diff
@@ -62,6 +62,9 @@ class Application < Rails::Application
     config.assets.paths << Rails.root.join('..', '..', 'vendor', 'assets', 'bower_components')
 
     config.time_zone = 'London'
+
+    # Missing translations should raise exceptions
+    config.action_view.raise_on_missing_translations = true
```

Comment:
Could this be avoided/minimised if we tested the Welsh scenarios in the same way we test the English stuff? I support raising an exception but, hopefully this doesn't banish the need to put an emphasis on the need to test Welsh scenarios as well.